File: executable/r/basic-error-bar.r
library(plotly)
p <- plotly(username='test-runner', key='9h29fe3l0x')

trace1 <- list(
  x = c(0, 1, 2), 
  y = c(6, 10, 2), 
  error_y = list(
    type = "data", 
    array = c(1, 2, 3), 
    visible = TRUE
  )
)
response <- p$plotly(trace0, kwargs=list(filename="basic-error-bar", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/node/basic-error-bar.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

var data = [
  {
    x: [0, 1, 2], 
    y: [6, 10, 2], 
    error_y: {
      array: [1, 2, 3], 
      type: "data", 
      visible: true
    }, 
    type: "scatter"
  }
];

plot_url = plotly.plot(data, function (err, msg) {
    console.log(msg);
});